esignet-core/src/main/java/io/mosip/esignet/core/dto/BindingOtpRequest.java

```diff
@@ -24,4 +26,6 @@ public class BindingOtpRequest {
     @NotNull(message = ErrorConstants.INVALID_OTP_CHANNEL)
     @Size(min = 1, message = ErrorConstants.INVALID_OTP_CHANNEL)
     private List<@OtpChannel String> otpChannels;
+
+    private String captcha;
```

Comment:
This will hold the captcha token, so I think it's apt to rename this as captchaToken

Reply:
Renamed to captchaToken